.github/workflows/conformance-e2e-v1.13.yaml

```diff
@@ -428,12 +428,10 @@ jobs:
             ./cilium-cli status --wait
             kubectl -n kube-system exec daemonset/cilium -- cilium status
 
-            ./cilium-cli connectivity test --datapath --collect-sysdump-on-failure \
-              --sysdump-hubble-flows-count=1000000 --sysdump-hubble-flows-timeout=5m \
-              --sysdump-output-filename "cilium-sysdump-${{ matrix.name }}-<ts>"
```

Comment:
Why dropping the hubble flows and sysdump extra arguments?

Reply:
The whole `--datapath` got removed, so we need to invoke only once `cilium-cli connectivity test`. All hubble params are in the invocation below.